docs/docusaurus/docs/core/validate_data/checkpoints/manage_checkpoints.md

```diff
@@ -1,15 +1,219 @@
 ---
 title: Manage Checkpoints
 ---
-
 import InProgress from '../../_core_components/_in_progress.md';
 
-<InProgress/>
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import StepRequestADataContext from '../../_core_components/common_steps/_request_a_data_context.md';
+
+Checkpoints provide the ability to group Validation Definitions and run them together with shared parameters and automated Actions.
+
+At runtime a Checkpoint can take in dictionaries that filter the Batches in each Validation Definition and modify the parameters of the Expectations that will be validated against them.  These will apply to every Validation Definition in the Checkpoint.  Therefore, the Validation Definitions grouped together in a Checkpoint should have Batch Definitions that accept the same criteria for filtering Batches, and the Expectation Suites in each Validation Definition should share a list of valid parameters as well.
+
+After a Checkpoint receives Validation Results from a Validation Definition it will execute a list of Actions.  These Actions will perform some task based on the validation results that were returned.  Actions can include things such as updating Data Docs with the new Validation Results or sending alerts when validations fail.  The Actions list will be executed once for each Validation Definition in a Checkpoint.
+
+If a Checkpoint's list of Actions is empty the Checkpoint can still be run and its validation results will be saved to the Data Context, but nothing else will automatically be done.
 
 ## Create a Checkpoint
 
-## Get a Checkpoint
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Import the GX library and the `Checkpoint` class:
+
+  ```python title="Python" name="/core/validate_data/checkpoints/_examples/create_a_checkpoint.py import statements"
+  ```
+
+2. <StepRequestADataContext/>.
+
+  In this example the variable `context` is your Data Context.
+
+3. [Get the Validation Definition](/core/validate_data/validation_definitions/manage_validation_definitions.md#get-a-validation-definition-by-name) [or Definitions](/core/validate_data/validation_definitions/manage_validation_definitions.md#get-validation-definitions-by-attributes) to add to the Checkpoint.
+
+  In this example the variable `validation_definitions` is a list object containing a single Validation Definition.
+
+4. Optional. Determine the Actions to add to the Checkpoint.
+
+  In this example the variable `actions` is a list containing two Actions.  The first updates your Data Docs with the results of the Validation Definition.  The second sends a Slack notification if any of the Expectations in the Validation Definition failed:
```

Comment:
```suggestion
In this example, the variable `actions` is a list of two actions. The first updates your Data Docs with the results of the Validation Definition. The second sends a Slack notification if any of the Expectations in the Validation Definition failed:
```
You're introducing some examples with colons and others without. Use the colons consistently.